src/project.ts

```diff
@@ -130,28 +131,45 @@ export class Project {
 	 * Load tiapp file
 	 *
 	 */
-	private loadTiappFile () {
+	private async loadTiappFile  () {
 		this.isTitaniumApp = false;
-		// tslint:disable-next-line:no-console
+		let error;
 		const rootPath = workspace.rootPath;
 		if (!rootPath) {
-			// tslint:disable-next-line:no-console
 			return;
 		}
 		const filePath = path.join(rootPath, TIAPP_FILENAME);
 		if (utils.fileExists(filePath)) {
+			this.isTitaniumApp = true;
 			const fileData = fs.readFileSync(filePath, 'utf-8');
 			const parser = new xml2js.Parser();
 			let json;
 			parser.parseString(fileData, (err, result) => {
-				if (!err) {
-					json = result;
+				if (err) {
+					error = new InteractionError(`Errors found in tiapp.xml \n ${err}`);
+					error.interactionChoices.push({
+						title: 'Open tiapp.xml',
+						run: async () => {
+							const file = path.join(workspace.rootPath, 'tiapp.xml');
+							const document = await workspace.openTextDocument(file);
+							const columnExp = /Column: (.*?)(?:\s|$)/g;
+							const column = columnExp.exec(err)[1];
```

Comment:
I think xml2js does tend to throw errors that match this regex, but it might be best to try and have some error handling around these calls returning null rather than the expected array return value